frontend/index.jsx

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 import {h, render} from 'preact'
+import 'preact/debug'
```

Comment:
What is this one about?

Reply:
Didn't mean to commit that sorry

Reply:
Just enables the react devtools extension for local

Reply:
Ah, cool!

Reply:
Well, it enables it globally here, so we should maybe only enable it for local if wanted, or keep it if wanted. I just generally keep it uncommitted in my tree like this.

Reply:
After all it only works when NODE_ENV=development anyway so I don't really see the harm

Reply:
Would you be okay with keeping it in that case?

Reply:
Yep, it sounds fine.